grid-proxy/tools/db/generate.go

```diff
@@ -397,37 +606,113 @@ func generateNodeGPUs(db *sql.DB) error {
 			contract:     i % 2,
 			id:           "0000:0e:00.0/1002/744c",
 		}
-		if _, err := db.Exec(insertQuery(&g)); err != nil {
-			panic(err)
+
+		GPUs = append(GPUs, objectToTupleString(g))
+	}
+
+	if err := insertTuples(db, node_gpu{}, GPUs); err != nil {
+		return err
+	}
+
+	fmt.Println("node GPUs generated")
+
+	return nil
+
+}
+
+func generateContracts(db *sql.DB) error {
+	billCount := 1
+	// Note: it's order senstive
+	if err := generateRentContracts(db, &billCount); err != nil {
```

Comment:
having to pass the billCount by reference works in our use case, but in general it's a really good idea unless it's needed, as a pointer creates the possibility for the value to not even be there.
i believe we can work around that, right?